src/neo/Persistence/RawView.cs

```diff
@@ -0,0 +1,33 @@
+using Neo.IO.Caching;
+using Neo.IO.Wrappers;
+using Neo.Ledger;
+using System;
+
+namespace Neo.Persistence
+{
+    /// <summary>
+    /// Provide a read-only <see cref="StoreView"/> for accessing directly from database instead of from snapshot.
+    /// </summary>
+    public class RawView : StoreView
```

Comment:
@erikzhang, I like the idea of `RawView`, this is a good organization difference that will make the difference.

Did you think about other name to it? EAhuahueuhaea Maybe RoView? (ReadyOnlyView or CurrentView). I do not know...
Raw is like basic, crude, fresh: https://www.thesaurus.com/browse/raw

Reply:
Renamed.